subsys/bluetooth/controller/hci_driver.c

```diff
@@ -401,7 +401,12 @@ static void recv_thread(void *p1, void *p2, void *p3)
 	ARG_UNUSED(p2);
 	ARG_UNUSED(p3);
 
+#if defined(CONFIG_BT_BUF_EVT_DISCARDABLE_COUNT)
+	static uint8_t hci_buffer[MAX(BT_BUF_RX_SIZE,
```

Comment:
I think MAX might be undefined here.

Reply:
I think an util.h include might be missing.

Reply:
`MAX(a, b)` is defined in `sys/util.h` which included by several of the included header files. One example: 
`hci_vs.h` includes `hci.h` which includes it. I can add this include explicitly if you prefer that.

Reply:
At least my sample fails to build because it's undefined.
If this file uses functionality from sys/util.h then it should include sys/util.h